docs-website-src/content/blockers.md

```diff
@@ -47,29 +52,41 @@ You can discover many of these issues by downloading `elevate-cpanel` and runnin
 * **cPanel is up to date**
   * You will need to be on a version mentioned in the "Latest cPanel & WHM Builds (All Architectures)" section at http://httpupdate.cpanel.net/
   * Mitigation: `/usr/local/cpanel/scripts/upcp`
-* **nameserver**
-  * cPanel provides support for a myriad of nameservers. (MyDNS, nsd, bind, powerdns). On RHEL 8 based distros, it is preferred that you always be on PowerDNS.
+* **Name Server**
+  * cPanel provides support for a myriad of name servers. (MyDNS, NSD, BIND, PowerDNS). On RHEL 8 based distributions, it is preferred that you always be on PowerDNS.
   * Mitigation: `/scripts/setupnameserver powerdns`
 * **MySQL**
-  * We will upgrade you automatically to MariaDB 10.6 during elevation if you do not do this in advance.
+  * If the version of MySQL/MariaDB installed on the system is not supported on RHEL 8 based distributions, it will need to be upgraded to a version that is. If the MySQL installation is managed by cPanel we will offer to upgrade MySQL automatically to MariaDB 10.6 during elevation. Declining the offer to upgrade MySQL will block the elevation. If the MySQL installation is managed by CloudLinux, then the upgrade must be performed manually.  This can be done by running the following command:
+  `/usr/local/cpanel/bin/whmapi1 start_background_mysql_upgrade version=10.6`
```

Comment:
I would leave this part out of the documentation here or at least clarify that it is only if MySQL is managed by cPanel.  

It reads like this is the command to execute to update MySQL managed by CloudLinux, but this is likely bad advice.  If it is managed by CloudLinux, then they would most likely want to use MySQL governor instead.  In that case, the command would look something like:

```
/usr/share/lve/dbgovernor/mysqlgovernor.py --mysql-version=mariadb106
/usr/share/lve/dbgovernor/mysqlgovernor.py --install
```

If anything, it is probably better to point them to this:  https://docs.cloudlinux.com/shared/cloudlinux_os_components/#upgrading-database-server

But my recommendation is to leave that advice out of the documentation since the script already has it when appropriate.